ebusd-2.1.x/de/vaillant/15.ctlv.SW0514.HW1104.csv

```diff
@@ -0,0 +1,322 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,70000,CTLV2,700 110,,,,,,,,,,
+
+# ##### Generell #####,,,,,,,,,,,,,
+*r,,,,,,B524,02000000,,,IGN:4,,,
+*w,,,,,,B524,02010000,,,,,,
+*wi,#install,,,,,B524,02010000,,,,,,
+r;w,,ContinuosHeatingMode,Durchgehendes Heizen,,,,0200,,,tempv,,,"-26=off, when the outside temperature falls below this threshold temperature the continuous heating function is started (off <=> function is disabled)"
```

Comment:
When using `temps2` as unit, negative values could be written via MQTT. Otherwise I received an error.
Typo regarding name: `ContinuousHeatingMode`